### ISSUE TITLE
[Request]: Slow export of bottles, maybe use multi threaded compressor?

### ISSUE THREAD

**Issue by lemonzest79** [bottlesdevs/Bottles#1461]
### Tell us the problem or your need

Last time I tried to export a bottle it took about 10 minutes for a 6.5GB I think bottle, the export only used one thread to create the tar.gz file maybe it would be better to use pigz? it can use many threads to compress, my system specs are below

CPU: AMD Ryzen 7 3800X 8c16t 3.9 - 4.5Ghz
Memory: 48GB
Home Drive: PCIe 3.0 4x NVME SSD

### Describe the solution you'd like

Some kind of multi threaded compress/decompressor like pigz or p7zip

### Other solutions?

Unknown

**Comment by mirkobrombin** [comment]
Thanks for reporting.

**Comment by lemonzest79** [comment]
yeah that sounds ok :) zstd as far as I know can be threaded to use more cpu cores
